services/cases-api/helpers/statusCheckCondition.js

```diff
@@ -1,6 +1,16 @@
-function isOngoing({ answers, people }) {
-  return answers && isEncrypted(answers) && !hasApplicantSigned(people);
-}
+import {
+  ACTIVE_ONGOING,
+  ACTIVE_SIGNATURE_PENDING,
+  ACTIVE_SIGNATURE_COMPLETED,
+  ACTIVE_SUBMITTED,
+  ACTIVE_COMPLETION_ONGOING,
+  ACTIVE_COMPLETION_SUBMITTED,
+  ACTIVE_RANDOM_CHECK_ONGOING,
+  ACTIVE_RANDOM_CHECK_SUBMITTED,
+} from '../../../libs/constants';
+
+const COMPLETION_REQUIRED = 'COMPLETION_REQUIRED';
```

Comment:
Is this the right place for these values ? `COMPLETION_REQUIRED` and `RANDOM_CHECK_REQUIRED` ?